packages/ui/ProgressBar/ProgressBar.js

```diff
@@ -121,7 +121,15 @@ const ProgressBar = kind({
 		 * @default 0
 		 * @public
 		 */
-		progressAnchor: PropTypes.number
+		progressAnchor: PropTypes.number,
+
+		/**
+		 * Display separator at the center
```

Comment:
Can you please update this to use the position of the `progressAnchor` instead of just center?  Particularly for `ui`, we can't assume it will only be center.

Reply:
The position of the separator has been modified to change according to the progressAnchor value